scylla/src/transport/cql_types_test.rs

```diff
@@ -16,23 +16,27 @@ use std::net::{IpAddr, Ipv4Addr, Ipv6Addr};
 use std::str::FromStr;
 
 // Used to prepare a table for test
-// Creates a new keyspace
+// Creates a new keyspace, without tablets if requested and the ScyllaDB instance supports them.
 // Drops and creates table {table_name} (id int PRIMARY KEY, val {type_name})
-async fn init_test(table_name: &str, type_name: &str) -> Session {
+async fn init_test_maybe_without_tablets(
+    table_name: &str,
+    type_name: &str,
+    should_disable_tablets: bool,
+) -> Session {
```

Comment:
I don't think the name change is needed, it doesn't help with anything.

It would be better to change parameter to `enable_tablets` to avoid unnecessary negation - it is easier for a human to reason about.

Reply:
What name change do you have in mind? Even though GH shows this as a name change, what I actually did here is introducing a new function. The old function's logic is delegated here. Thanks to that, we don't have to do a huge lot of name changes.

Reply:
Right, I didn't notice that `init_test` still exists.

The part about negated semantics still stands. It's actually the same point you made in review of my PR that introduced tablets: https://github.com/scylladb/scylla-rust-driver/pull/937#discussion_r1549144940